ApollosProject/apollos-apps
# fix(prayer): add trim to input to prevent empty submission

## ✏️ Description

- adds checks for empty or whitespace input to prevent empty prayer submissions



## 🔬 To Test
1. Open up the Prayer Experience
2. Press "done" in the upper right corner to navigate to the Prayer Input screen
3. Tap "I need prayer for..."
4. Without entering any text, press the "done" key on your SOFTWARE KEYBOARD
5. Press should not submit prayer and share button should still be disabled

## 📸 Screenshots


- whitespace input with disabled share button
![image](https://user-images.githubusercontent.com/7557597/205160584-e7d92148-2615-4bd3-8cca-dd299695cfbb.png)


## ⚠️ To-do Before Merge



## 📝 Checklist
